Doc/library/doctest.rst

```diff
@@ -381,12 +398,20 @@ The fine print:
 What's the Execution Context?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, each time :mod:`doctest` finds a docstring to test, it uses a
-*shallow copy* of :mod:`M`'s globals, so that running tests doesn't change the
-module's real globals, and so that one test in :mod:`M` can't leave behind
-crumbs that accidentally allow another test to work.  This means examples can
-freely use any names defined at top-level in :mod:`M`, and names defined earlier
-in the docstring being run. Examples cannot see names defined in other
+Within a docstring, later examples can use names defined by earlier
+examples. It's fine for an example to set up state, and
+have no output. 
+
+For each docstring, :mod:`doctest` makes (by default) a 
+*shallow copy* of :mod:`M`'s globals.
+This means examples can freely use any names defined at the top level of 
+:mod:`M`. 
+When doctest tests examples, it doesn't change the module's real globals.
+
+This shallow copy of globals is discarded at the end of each docstring,
+and copied afresh for the next docstring. Thus, examples in one docstring
+in :mod:`M` can't leave behind crumbs that accidentally allow an example
+in another docstring to work.  Examples cannot see names defined in other
 docstrings.
```

Comment:
Thank you.  This is another instance of me working out my frustrations against the old text.